src/providers/ad/ad_gpo.c

```diff
@@ -3435,7 +3435,7 @@ ad_gpo_get_som_attrs_done(struct tevent_req *subreq)
         goto done;
     }
     if ((num_results < 1) || (results == NULL)) {
-        DEBUG(SSSDBG_OP_FAILURE, "no attrs found for SOM; try next SOM.\n");
+        DEBUG(SSSDBG_MINOR_FAILURE, "no attrs found for SOM; try next SOM.\n");
```

Comment:
Does it mean "No GPO was found for given scope of management (SOM)"?

Reply:
IIUC yes